src/sql/workbench/contrib/notebook/browser/outputs/plotlyOutput.component.ts

```diff
@@ -137,6 +139,12 @@ export class PlotlyOutputComponent extends AngularDisposable implements IMimeCom
 		// No need to re-layout for now as Plotly is doing its own resize handling.
 	}
 
+	private resize() {
+		if (this._initialized) {
```

Comment:
This doesn't seem necessary - you only hook up the resize listener after initialized is set to true so this doesn't seem like it'd ever be false.